### ISSUE TITLE
Split sections into tabs/filters

### ISSUE THREAD

**Issue by rafaeldamasceno** [axiel7/AniHyou-iOS#42]
**Request (required)**
When clicking on Home, Anime or Manga I expect to see some content directly (like on the profile tab). The fact that they are just menus for the actual sections of content makes for a frustrating UX.

I would suggest to split Discover and Activity into their own tabs (they aren’t really similar). Maybe Discover could be merged with Search, showing the current Discover content after the existing buttons on the Search page. I don’t know where Notifications would fit best. 

As for Anime and Manga, my suggestion would be to make the existing section into a filter (so the tab would show directly some content, maybe Watching by default, but with a dropdown in the title to select other categories). Another possible solution (that I really dislike) would be to make it into an accordion.

**Examples**
AniTrack has Explore as a Discover/Search combination with a button to toggle anime and manga. Home only shows activity with a way to filter what kind of activity. Anime/Manga tabs are organized by accordions (again, I would suggest filters instead of this).

AniHyou Android immediately shows content on Home with top navigation between Activity and Discover.

**Comment by axiel7** [comment]
Most native iOS apps work like that. The last opened section is always saved so you see the list of sections only the first time you install the app.

**Comment by rafaeldamasceno** [comment]
I don’t agree. Take the Phone app, the Music app. You might be 5 submenus down inside the content, you can always press the navigation bar to return to the main page for that tab.

Also, given that Home host 3 very different pages in this app, I do want to switch frequently and the fact that the navigation bar doesn’t help me go back to the initial context is super cumbersome for UX.

**Comment by rafaeldamasceno** [comment]
Good job on splitting the explore section from the home tab on the latest update!

It would be super nice if you also now also split activity and current, since a home tab which has two submenus is pretty much useless. There is never a moment where I see this screen and want to be there:

<img width="1284" height="2778" alt="Image" src="https://github.com/user-attachments/assets/56983611-df4d-4395-a3c5-b7632c600161" />

It is void of functionality and information. 

Either do something like this:

<img width="727" height="349" alt="Image" src="https://github.com/user-attachments/assets/3452aa4f-7b12-4a0c-98cf-8e46f15b03aa" />

Or just another tab selector at the top…

<img width="783" height="1449" alt="Image" src="https://github.com/user-attachments/assets/af09f465-b7f1-49cd-b48e-e360be423d55" />
